part7/anecdotes/src/App.js

```diff
@@ -1,12 +1,36 @@
 import React, { useState } from 'react'
 import { Switch, Route, useRouteMatch } from 'react-router-dom'
+
 import Menu from './components/Menu'
 import AnecdoteList from './components/AnecdoteList'
 import Anecdote from './components/Anecdote'
 import About from './components/About'
 import CreateNew from './components/CreateNew'
 import Footer from './components/Footer'
 
+import styled from '@emotion/styled'
+import { css } from '@styled-system/css'
+import Layout from './design/layout'
+
+// Es nesaprotu, kā lai tiek vaļā no tā <body> margin 
+
+// const Box = styled.body({
+//   boxSizing: 'border-box',
+//   m: 0,
+//   minWidth: 0,
+// })
```

Comment:
Palasi par [Emotion Global](https://emotion.sh/docs/globals) stiliem. Ieteicams likt kā siblingu `<App>`. 

`<body>` netiek definēts ar React, līdz ar to, tu viņu kā komponenti nekad no React neizmantosi.

Reply:
par šo mana izpēte ir ieilgusi